languages/go/README.md

```diff
@@ -10,7 +10,9 @@ managing projects and secrets, as well as a client interface to facilitate opera
 
 ## Installation
 
-Download the SDK files and place them in your Go project directory.
+You can install the [sdk-go](https://github.com/bitwarden/sdk-go) Go module like any other Go module (for example, with `go get`). You will just need to make sure you can build with [cgo](https://pkg.go.dev/cmd/cgo) on your machine.
+
+You can find instructions [here](./INSTRUCTIONS.md).
```

Comment:
Should we just link to the installation instructions and any relevant information be housed there? 

I'm on the fence, so don't feel pressure to follow this suggestion.

```
Follow the installation instructions [here](./INSTRUCTIONS.md).
```

Reply:
That's a good idea, it keeps all the relevant info in one place.

Fixed in https://github.com/bitwarden/sdk/pull/861/commits/39398b057aa7542f0a75efb7e6936195196fc5ae.